src/defaultsettings.cpp

```diff
@@ -224,6 +225,8 @@ void set_default_settings(Settings *settings)
 	settings->setDefault("font_size", fontsize.str());
 	settings->setDefault("mono_font_size", fontsize.str());
 #endif
+	//Main menu
+	settings->setDefault("enable_modmanager", "false");
```

Comment:
Same here. This is the wrong place, it's a client setting. Put it next to the other main menu settings (such as mainmenu_last_idx or something)